doc/Usage.adoc

```diff
@@ -0,0 +1,272 @@
+= Usage
+
+:toc:
+:toc-placement!:
+
+The following sections explain how you can use the Landrush DNS server and customize it as per your requirements, once you have link:README.adoc[installed] it.
+
+'''
+toc::[]
+'''
+
```

Comment:
What would be nice in this guide would be a section "Configuration options" which contains a table listing all the various options mentioned across this guide, a short sentence of its purpose and a link to the section where you find more information. This gives a quick overview what's configurable. I've seen several issues which originate form the fact that people did not know that a certain config option was available.